server/routes/induction/update/previousWorkExperienceDetailUpdateController.test.ts

```diff
@@ -7,19 +7,14 @@ import InductionService from '../../../services/inductionService'
 import PreviousWorkExperienceDetailUpdateController from './previousWorkExperienceDetailUpdateController'
 import aValidPrisonerSummary from '../../../testsupport/prisonerSummaryTestDataBuilder'
 import { aLongQuestionSetInductionDto } from '../../../testsupport/inductionDtoTestDataBuilder'
-import validatePreviousWorkExperienceDetailForm from './previousWorkExperienceDetailFormValidator'
 import toCreateOrUpdateInductionDto from '../../../data/mappers/createOrUpdateInductionDtoMapper'
 import { aLongQuestionSetUpdateInductionDto } from '../../../testsupport/updateInductionDtoTestDataBuilder'
 import TypeOfWorkExperienceValue from '../../../enums/typeOfWorkExperienceValue'
 
-jest.mock('./previousWorkExperienceDetailFormValidator')
 jest.mock('../../../data/mappers/createOrUpdateInductionDtoMapper')
 jest.mock('../../../services/inductionService')
 
 describe('previousWorkExperienceDetailUpdateController', () => {
-  const mockedFormValidator = validatePreviousWorkExperienceDetailForm as jest.MockedFunction<
```

Comment:
As with the other controller tests that I've been updating recently, removing the use of a mock for the form validator, and using the real validator instead